src/commands.rs

```diff
@@ -398,8 +398,16 @@ implement_commands! {
     }
 
     /// Get the value of a key and set expiration
-    fn get_ex<K: ToRedisArgs>(key: K, seconds: usize) {
-        cmd("GETEX").arg(key).arg("EX").arg(seconds)
+    fn get_ex<K: ToRedisArgs>(key: K, expire_at: ExpiryOption) {
+        let (option, time_arg): (_, Option<usize>) = match expire_at {
```

Comment:
Is the type annotation necessary here? I would expect it isn't...

Reply:
It isn't needed. I forgot to remove that. I will update that.

Reply:
Done!